docs/python/README.md

```diff
@@ -0,0 +1,378 @@
+
+> [!NOTE]
+> Skip ahead if you want to build the application first and get into the details later. You won't need to
+> know about builders for the rest of this tutorial.
+
+In short, a builder is a delivery mechanism for buildpacks. A builder contains references to base images and individual buildpacks. A base image contains the operating system and system dependencies. Buildpacks are the components that will configure an image to run your application, that’s where the bulk of the logic lives and why the project is called “Cloud Native Buildpacks” and not “Cloud Native Builders.”
+
+You can view the contents of a builder via the command `pack builder inspect`. For example:
+
+```
+$ pack builder inspect heroku/builder:24 | grep Buildpacks: -m1 -A10
+Buildpacks:
+  ID                                NAME                               VERSION        HOMEPAGE
+  heroku/deb-packages               Heroku .deb Packages               0.0.3          https://github.com/heroku/buildpacks-deb-packages
+  heroku/dotnet                     Heroku .NET                        0.1.9          https://github.com/heroku/buildpacks-dotnet
+  heroku/go                         Heroku Go                          0.5.0          https://github.com/heroku/buildpacks-go
+  heroku/gradle                     Heroku Gradle                      6.0.4          https://github.com/heroku/buildpacks-jvm
+  heroku/java                       Heroku Java                        6.0.4          https://github.com/heroku/buildpacks-jvm
+  heroku/jvm                        Heroku OpenJDK                     6.0.4          https://github.com/heroku/buildpacks-jvm
+  heroku/maven                      Heroku Maven                       6.0.4          https://github.com/heroku/buildpacks-jvm
+  heroku/nodejs                     Heroku Node.js                     3.4.0          https://github.com/heroku/buildpacks-nodejs
+  heroku/nodejs-corepack            Heroku Node.js Corepack            3.4.0          https://github.com/heroku/buildpacks-nodejs
+```
+<!-- STOP. This document is autogenerated. Do not manually modify. See the top of the doc for more details. -->
+
+> [!NOTE]
+> Your output version numbers may differ.
+
+This output shows the various buildpacks that represent the different languages that are supported by this builder such as `heroku/go` and `heroku/nodejs-engine`.
+## Download an example Django application
+
+How do you configure a CNB? Give them an application. While Dockerfile is procedural, buildpacks, are declarative. A buildpack will determine what your application needs to function by inspecting the code on disk.
+
+For this example, we're using a pre-built Django application. Download it now:
+
+```
+$ git clone https://github.com/heroku/python-getting-started
+$ cd python-getting-started
+```
+<!-- STOP. This document is autogenerated. Do not manually modify. See the top of the doc for more details. -->
+
+Verify you're in the correct directory:
+
+```
+$ ls
```

Comment:
One of the key Python files is `.python-version`, which doesn't show up with `ls` since it's a dot-file. Whilst the purpose of the `ls` is more to check one is on the correct directory, I think it might be best to run `ls -A` instead, which shows dot-files (but unlike `ls -a` doesn't show the `.` and `..` entries).